src/AdjustSync.cpp

```diff
@@ -358,15 +380,20 @@ void AdjustSync::GetSyncChangeTextSong( std::vector<RString> &vsAddTo )
 			}
 		}
 
-		const std::vector<TimingSegment*> &bpmTest = testing.GetTimingSegments(SEGMENT_BPM);
-		const std::vector<TimingSegment*> &bpmOrig = original.GetTimingSegments(SEGMENT_BPM);
+		// Reserve space in the vector
+		vsAddTo.reserve(vsAddTo.size() + 100);
```

Comment:
We can calculate how much we need to reserve beforehand right instead of a flat +100?

Reply:
> We can calculate how much we need to reserve beforehand right instead of a flat +100?

Hmm, I could sum the `bpmTest`, `stopTest`, and `delyTest` but that might end up negating any savings on performance given by reserving space in the vector. We could just revert this and let it reallocate memory like it used to. It's not like it was being a problem there.